backend/schema/corpora_schema.md

```diff
@@ -0,0 +1,323 @@
+# Corpora Schema
```

Comment:
Are we deprecating Corpora here?

Reply:
Let's punt on this for the time being. We'll need to ask @ambrosejcarr what the new word is for the data integration work part of the project now known as cellxgene.